use-timescale/migration/about-hypershift.md

```diff
@@ -0,0 +1,119 @@
+
+# About Hypershift
+
+<HypershiftIntro />
+
+## Migrating from a managed database
+
+If you're migrating from an Amazon RDS database, the best option is to run
+hypershift on an EC2 instance in the same availability zone as your RDS
+instance.
+
+<Highlight type="note">
+Availability zones are assigned different names in different AWS
+accounts. To make sure that you are using the same zone refer to
+[AWS user guide](https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html)
+</Highlight>
+
+For migrations from other managed services, including Managed Service for
+TimescaleDB, run hypershift from a virtual machine in the same region as either
+the source or target database, whichever is most convenient.
+
+## Migration speed
+
+In preliminary testing, hypershift migrated 60&nbsp;GB of data in 9 to 12
+minutes, and 1&nbsp;TB of data in under 4 hours.
+
+When you run hypershift, it uses compute resources on the machine that you run
+the command on to copy data from the source to the target database. This means
+that the network throughput and latency that you have between the various
+machines can impact the speed and duration of the migration.
+
+You can continue reading from your source database during the migration, though
+performance could be slower. If you write to tables in your source database
+during the migration, the new writes are not transferred to Timescale Cloud. To
+avoid this problem, fork your database and migrate your data from the fork.
+
+<Highlight type="important">
+If you have a large database, and hypershift is going to have to run for a very
+long time to migrate it, for example, a day or more, ensure that you have a
+stable network connection. Hypershift is not able to recover if the network
+connection is interrupted. Additionally, when Hypershift is running, it holds a
+single transaction open for the entire duration of the migration. This prevents
+any automated vacuum tasks from running, which can cause a range of different
+problems on a busy source database.
+</Highlight>
+
+## The hypershift command line tool
+
+Hypershift is launched from the command prompt. You can provide a number of
+different options to hypershift when you issue the command. This section
+describes the various options.
+
+You can also provide configuration to hypershift using a YAML configuration
+file. For more information about constructing a hypershift configuration file,
+see the
+[hypershift configuration section][hypershift-config].
+
+You can provide the source and target database URIs using either command line
+flags, or the configuration file. If the information is provided in both,
+hypershift uses the URIs provided in the command line flags, and ignores the
+URIs in the configuration file.
+
+To see all available configuration options for hypershift, use this command:
+
+```bash
+docker run -ti timescale/hypershift:0.4 --help
+```
+
+Within the Docker container, use this command:
```

Comment:
I'm not sure that showing the execution in the docker container is useful. I doubt that many users will expect to run it this way.